docs/marketing/user-guide/part-5-practical-workflows.md

```
@@ -23,10 +23,10 @@ Follow this sequence for new brands:
 
 #### Step 1: Brand Context Creation
 
-Start with the paw-mkt-agency skill:
+Start with the paw-mkt-agent-agency skill:
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)